backend/tests/unit/scrape/agencyShortName.test.js

```diff
@@ -0,0 +1,34 @@
+const each = require('jest-each').default;
+
+const agencyType = require("scrape/agencyShortName");
+
+const stub = [
+    "stub",
+    "test",
+    "testing",
+]
+
+each([undefined, null]).test('return "Unknown" on undefined/null', (agencyID) => {
+   
+
+    const result = agencyType(agencyID, stub)
+
+    expect(result).toBe('Unknown');
+});
+
+
+each([-100, 0, 100]).test('return "Unknown" on value outside reference table', (agencyID) => {
```

Comment:
How about adding 4 here, just to be safe?

Reply:
Sounds reasonable.